services/convert/notification.go

```diff
@@ -71,7 +72,7 @@ func ToNotificationThread(ctx context.Context, n *activities_model.Notification)
 		url := n.Repository.HTMLURL() + "/commit/" + url.PathEscape(n.CommitID)
 		result.Subject = &api.NotificationSubject{
 			Type:    api.NotifySubjectCommit,
-			Title:   n.CommitID,
+			Title:   strings.TrimSpace(n.Commit.CommitMessage),
```

Comment:
Is it fine to display the commit's full message to end users? Even if the message contains thousands of words?